tests/e2e/e2e_multiple_services_per_pod_test.go

```diff
@@ -0,0 +1,142 @@
+package e2e
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	. "github.com/openservicemesh/osm/tests/framework"
+)
+
+var _ = OSMDescribe("Test access via multiple services matching the same pod",
+	OSMDescribeInfo{
+		Tier:   2,
+		Bucket: 2,
```

Comment:
I doubt a newcomer to the repo would know what this is for. At some point it would be great to add a comment (to each one) to explain what that is for and how it works.

Reply:
These are explained in the type definition. Not sure it makes sense to duplicate every place they're used.
https://github.com/openservicemesh/osm/blob/87e6c977726289b5b21f0c9abe69b86a82b04242/tests/framework/common.go#L111-L119

Reply:
This is well documented in the e2e readme: https://github.com/openservicemesh/osm/blob/main/tests/e2e/README.md#files-and-structure